website/views/options/inventory/time-travelers.jade

```diff
@@ -1,35 +1,29 @@
 .container-fluid
-  .stable.row(ng-if='user.purchased.plan.consecutive.trinkets <= 0')
-    .col-md-2
-      .npc_timetravelers
+  .stable.row(ng-if='user.purchased.plan.consecutive.trinkets <= 0'): .col-xs-12
+    .npc_timetravelers.pull-left-sm.col-centered
+    .popover.static-popover.fade.right.in.pull-left-sm.col-centered
+      .arrow.hidden-xs
+      h3.popover-title!=env.t('timeTravelersTitleNoSub', {linkStartTyler: "<a href='https://github.com/lefnire' target='_blank'>", linkStartVicky: "<a href='http://blog.habitrpg.com/who' target='_blank'>", linkEnd: "</a>"})
+      .popover-content
+        p!=env.t('timeTravelersPopoverNoSub', {linkStart: "<a href='https://habitrpg.com/#/options/settings/subscription' target='_blank'>", linkEnd: "</a>"})
 
-    .col-md-10
-      .popover.static-popover.fade.right.in
-        .arrow
-        h3.popover-title!=env.t('timeTravelersTitleNoSub', {linkStartTyler: "<a href='https://github.com/lefnire' target='_blank'>", linkStartVicky: "<a href='http://blog.habitrpg.com/who' target='_blank'>", linkEnd: "</a>"})
-        .popover-content
-          p!=env.t('timeTravelersPopoverNoSub', {linkStart: "<a href='https://habitrpg.com/#/options/settings/subscription' target='_blank'>", linkEnd: "</a>"})
+  .stable.row(ng-if='user.purchased.plan.consecutive.trinkets > 0'): .col-xs-12
+    .npc_timetravelers_active.pull-left-sm.col-centered
+    .popover.static-popover.fade.right.in.pull-left-sm.col-centered
+      .arrow.hidden-xs
+      h3.popover-title=env.t('timeTravelersTitle')
+      .popover-content
+        .pull-right
+          span.inventory_special_trinket.inline-gems
+          b x{{user.purchased.plan.consecutive.trinkets}}
+        p!=env.t('timeTravelersPopover', {linkStart: "<a href='http://habitrpg.wikia.com/wiki/Mystery_Item' target='_blank'>", linkEnd: "</a>"})
 
-  .row.stable(ng-if='user.purchased.plan.consecutive.trinkets > 0')
-    .col-md-2
-      .npc_timetravelers_active
-
-    .col-md-10
-      .popover.static-popover.fade.right.in
-        .arrow
-        h3.popover-title=env.t('timeTravelersTitle')
-        .popover-content
-          .pull-right
-            span.inventory_special_trinket.inline-gems
-            b x{{user.purchased.plan.consecutive.trinkets}}
-          p!=env.t('timeTravelersPopover', {linkStart: "<a href='http://habitrpg.wikia.com/wiki/Mystery_Item' target='_blank'>", linkEnd: "</a>"})
-
-    .col-md-12
-      li.customize-menu.inventory-gear
-        menu.pets-menu(label='{{::set.text}}', ng-repeat='set in Content.timeTravelerStore(user.items.gear.owned)')
-          div(ng-repeat='item in set.items')
-            button.customize-option(class='shop_{{::item.key}}',
-              popover='{{::item.notes()}}', popover-title='{{::item.text()}}',
-              popover-trigger='mouseenter', popover-placement='right',
-              popover-append-to-body='true',
-              ng-click='user.ops.buyMysterySet({params:{key:set.key}})')
+  .row: .col-md-12
```

Comment:
@crookedneighbor: I now understand the difference between this and `.row.col-md-12`, but I don't quite get why one would be preferred over the other. Functionally, it doesn't seem to make a difference (at least I couldn't find one in the testing I did). Is it because it's just better practice?